<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_server">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>server</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IP">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>118.89.104.130</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>port</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>8083</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_findDevice">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>查询设备状态</string>
    </property>
   </widget>
   <widget class="Line" name="line_head">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_deviceState">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::CustomDashLine</enum>
    </property>
    <row>
     <property name="text">
      <string>imei号</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>版本号</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>状态</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>时间</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>经纬度</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>GSM信号</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>电压值</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>方向</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>速度</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>数据</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_FindDeviceLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>查询设备日志</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IMEI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>865067023193645</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_findLog">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>411</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_GPS">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_GSM">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GSM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LOG">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LOG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SETTING">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_BATTERY">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>battery</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_REBOOT">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>reboot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UPGRADE">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>upgrade</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
